common/md5.h

```diff
@@ -27,6 +27,15 @@
 
 namespace Common {
 
+/**
+ * @defgroup common_md5 MD5 checksum
+ * @ingroup common
+ *
+ * @brief API for computing the MD5 checksum.
```

Comment:
This is less of an API and more of a method/class - @sev- WDYT on terminology in general

Reply:
That's a good point about having consistent terminology. As I see it, 'API' is a superset of whatever we might have in these header files - classes, methods, functions, structures etc. If you agree with this, then probably using the term 'API' is safest and most future-proof.

Reply:
In this specific case I would probably have used `Functions` if I had written the documentation. But API is also fine with me. And indeed we need consistency.